src/jug.sol

```diff
@@ -22,14 +22,14 @@ contract Jug is LibNote {
 
     // --- Data ---
     struct Ilk {
-        uint256 duty;
-        uint256  rho;
+        uint256 duty;  // Stability Fee      [ray]
```

Comment:
Just labelling this "Stability Fee" is dangerous because it really isn't the stability fee, that's `base + duty`, and if both of those values are non-zero, the non-linear correspondence between addition of per-second rates and the effective yearly rate creates a lot of risk of confusion. I know it's more verbose, but something like "collateral-specific per-second stability fee contribution" is likely to be more robustly understood.

Reply:
I second this, a number of people have already stumbled over this detail.

Reply:
Completely agree. Later on, the glossary should be updated with the description we adopt here. 
I think the benefit in comprehension outweighs the cost of verbosity. 

I like "Collateral-specific, per-second stability fee contribution".

Definitely worth another discussion, but I think `base` should be set to `1` and `duty` is set to the collateral-specific component amount; any system-wide stability fee that's enforced can be contained within `base` (e.g. `1.00000000XYZ`). It's much easier to explain to integration partners (had to add some nuance on this earlier today) and seems to be the intended method of implementing rates.

Reply:
Go ahead with the verbose description but let's avoid in-depth discussion of what the best convention for using `base` and `duty` is in this PR ;). (There's a not-unreasonable counter-argument that says `base` should be a fractional value and `duty` should be `>= ONE` based on the fact that `init` sets `duty` to `ONE`, but this is no place to bikeshed on that...the meta-point is that only `base+duty` has an unambiguous interpretation. And I'm personally a fan of "`base` is superfluous, shouldn't be used at all, and in fact shouldn't exist in future versions of the system".).

Reply:
Sounds good. I'll go with "Collateral-specific, per-second stability fee contribution".

I assume you're aware, but for record, it's important that any changes (to how rates are enforced today) are shared with INTG team, so we can push outward to our network 👍 . Thanks again.